test/dotnet-core-uninstall.Tests/Shared/Utils/RegexesTests.cs

```diff
@@ -68,103 +47,6 @@ internal void TestBundleMajorMinorRegexReject(string input)
             TestRegexReject(Regexes.BundleMajorMinorRegex, input);
         }
 
-        [Theory]
-        [InlineData("Microsoft .NET Core SDK 2.2.202 (x64)")]
-        [InlineData("Microsoft .NET Core SDK 2.1.300 - rc1 (x86)")]
-        [InlineData("Microsoft .NET Core SDK 3.0.100 - preview5 (x64)")]
-        [InlineData(".NET Core SDK 1.1.10 (x64)")]
-        [InlineData("Microsoft .NET Core SDK - 2.1.4 (x86)")]
-        [InlineData("Microsoft .NET Core Runtime - 2.2.4 (x64)")]
-        [InlineData("Microsoft .NET Core Runtime - 2.1.0 Release Candidate 1 (x86)")]
-        [InlineData("Microsoft .NET Core Runtime - 3.0.0 Preview 5 (x64)")]
-        [InlineData("Microsoft .NET Core 1.1.13 - Runtime (x64)")]
-        [InlineData("Microsoft ASP.NET Core 3.0.0 Preview 6 Build 19307.2 - Shared Framework")]
-        [InlineData("Microsoft ASP.NET Core 2.2.6 - Shared Framework")]
-        [InlineData("Microsoft ASP.NET Core 2.2.0 Preview 3 Build 35497 - Shared Framework")]
-        [InlineData("Microsoft ASP.NET Core 2.1.0 Release Candidate 1 - Shared Framework")]
```

Comment:
why these tests are removed? due to corescan logic does not support it?

Reply:
Half of the deleted tests are moved [here](https://github.com/dotnet/cli-lab/pull/170/files#diff-b36d4685ade1f3d7383ef3ff9de9d9233eb115cec27950ba348518fc290c4ed5R13) since they are no longer directly testing the regex. The other half are removed because corescan's logic is less restrictive about which bundles are considered .NET bundles. With the regex's we were looking for very specific display name strings and with the corescan logic we're less exact.

For example, out original logic might not have marked `Microsoft ASP.NET Core 2.1.0 Release Candidate 1 - Shared Framework` as a .NET bundle because there are extra spaced around the `-`. The corescan logic doesn't care about that kind of thing, which is good because it will be more resillient to small format changes.